owncloudApp/src/main/java/com/owncloud/android/ui/activity/LogHistoryActivity.java

```diff
@@ -184,10 +177,10 @@ public boolean onMenuItemActionExpand(MenuItem item) {
             });
         }
 
-        if (null != searchView) {
-            searchView.setSearchableInfo(searchManager.getSearchableInfo(getComponentName()));
-            searchView.setIconifiedByDefault(true);
-            searchView.setOnQueryTextListener(queryTextListener);
+        if (null != mSearchView) {
```

Comment:
And if you keep `if (mSearchView == null) { return true }` in https://github.com/owncloud/android/pull/2440/commits/d2520555466d395603d89e6ba0303acb9ffe2465#diff-4100ef3dfbd51a1d654425db433334c2L141, you would not need to check it here again

Reply:
The first way was fine too.
Done